components/widgets/team_icon/team_icon.scss

```
@@ -34,7 +34,7 @@
     width: 34px;
     text-align: center;
     text-transform: uppercase;
-    background-color: rgba(255, 255, 255, 0.16);
+    background-color: var(--sidebar-text-16);
```

You missed a few more of these
```suggestion
    background-color: rgba(var(--sidebar-text-rgb), 0.16);
```